sql/src/main/java/io/crate/analyze/ShowStatementAnalyzer.java

```diff
@@ -77,6 +79,51 @@ AnalyzedStatement analyzeShowTransaction(Analysis analysis) {
         return newAnalysis.analyzedStatement();
     }
 
+    Query rewriteShowSessionParameter(ShowSessionParameter node) {
+        /*
+         * Rewrite
+         * <code>
+         *     SHOW { parameter_name | ALL }
+         * </code>
+         * To
+         * <code>
+         *     SELECT [ name, ] setting
+         *     FROM pg_catalog.pg_settings
+         *     [ WHERE name = parameter_name ]
+         * </code>
+         */
+        StringBuilder sb = new StringBuilder("SELECT ");
+        QualifiedName sessionSetting = node.parameter();
+        if (sessionSetting != null) {
+            sb.append("setting ");
+        } else {
+            sb.append("name, setting ");
```

Comment:
nitpick: can we add the description as well when using the `all` option and since we now have this info as well? pg does that

Reply:
yep, shall it go to this pr then? would probably open a small pr for it later, coz it is a user facing change.

Reply:
I don't see why not doing this here, it's a change for major release. 
Unless we want to backport it to 3.3 but I don't think that's the case, we are not fixing a bug here.

Personally, I'd do it here but no strong opinion.
@seut your opinion?

Reply:
I'd prefer at least a dedicated commit for that (including related CHANGES entry + docu). Dedicated PR is also fine for me.

Reply:
I'll go for a dedicated PR once this one is merged.